scripts/sandbox.ts

```diff
@@ -339,12 +365,14 @@ async function main() {
         debug: true,
       }
     );
-  } else {
+  } else if (build) {
     await executeCLIStep(steps.build, { cwd, dryRun, debug });
     // TODO serve
   }
 
   // TODO start dev
 }
 
-main().catch((err) => console.error(err));
+if (require.main === module) {
+  main().catch((err) => console.error(err));
```

Comment:
Now that sandbox is consumed by other scripts, you'll have to rethrow the error or exit with code 1 else it will get swallowed and it will seem like the command worked even though there was internal failure

Reply:
From there you can decide whether to bail on first failure, or to continue and collect which ones failed to output later